test/fixtures/condition/condition-greater-than.hbs

```diff
@@ -0,0 +1 @@
+{{#gt noListItems 1}}<h2>Greater</h2>{{/gt}}
```

Comment:
Do we need a dedicated custom helper for this?
In Muban we have https://github.com/mediamonks/muban/blob/master/build-tools/handlebars-helpers/condition.js, where you can use all condition types. Does `>` work in `htl` in a `-test` check?